app/views/layouts/base.html.erb

```diff
@@ -61,29 +61,16 @@
   <div class="usa-overlay"></div>
   <%= yield(:mobile_nav) if content_for?(:mobile_nav) %>
   <%= render 'shared/banner' %>
-  <div aria-label="main-navigation" role="navigation">
-    <% if content_for?(:nav) %>
-      <%= yield(:nav) %>
-    <% else %>
-      <% if decorated_session.sp_name %>
-        <%= render 'shared/nav_branded' %>
-      <% else %>
-        <%= render 'shared/nav_lite' %>
-      <% end %>
-    <% end %>
-  </div>
-  <main class="site-wrap bg-light-blue" id="main-content">
-    <div class="container">
-      <div class="padding-x-2 padding-y-4 tablet:padding-y-8 tablet:padding-x-10 margin-x-auto tablet:margin-bottom-8 border-box <%= local_assigns[:disable_card].present? ? '' : 'card' %>">
-        <%= yield(:pre_flash_content) if content_for?(:pre_flash_content) %>
-        <%= render FlashComponent.new(flash: flash) %>
-        <%= content_for?(:content) ? yield(:content) : yield %>
-      </div>
+<main class="site-wrap bg-light-blue" id="main-content">
+  <div class="container">
+    <div class="padding-x-2 padding-y-4 tablet:padding-y-8 tablet:padding-x-10 margin-x-auto tablet:margin-bottom-8 border-box <%= local_assigns[:disable_card].present? ? '' : 'card' %>">
+      <%= yield(:pre_flash_content) if content_for?(:pre_flash_content) %>
+      <%= render FlashComponent.new(flash: flash) %>
+      <%= content_for?(:content) ? yield(:content) : yield %>
     </div>
-  </main>
-
+  </div>
+</main>
```

Comment:
Can we maintain the indentation that existed previously?

Reply:
Ah, I thought that was an oversight. This should be addressed now.

Reply:
I still see it without any indentation 🤔

Reply:
That is weird. I went to view the change, but it does not find the commits. I think it got messed up when I did a force push--I'll find out what happened.